<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvrAddrConfigDialog</class>
 <widget class="QDialog" name="SvrAddrConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,1">
   <item>
    <widget class="QGroupBox" name="groupBoxSvr1">
     <property name="title">
      <string>Server1</string>
     </property>
     <widget class="QLabel" name="labelSvr1Addr">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSvr1Port">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSvr1Addr">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSvr1Port">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSvr2">
     <property name="title">
      <string>Server2</string>
     </property>
     <widget class="QLineEdit" name="lineEditSvr2Addr">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>11</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSvr2Addr">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>11</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSvr2Port">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>37</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSvr2Port">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>37</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
